plugin.json

```diff
@@ -2,7 +2,7 @@
   "id": "com.mattermost.mailermost-plugin",
   "name": "Mailermost",
   "description": "Replying to notification emails sends the reply as a post in Mattermost.",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
Please don't bump the version in this PR.

Reply:
@zarej let us know if you have questions about the above. It looks like this is otherwise good to merge :)

Reply:
Added new commit ;)